crates/runtime/src/accelerated_table/refresh_task/changes.rs

```diff
@@ -88,6 +94,12 @@ impl RefreshTask {
                             }
                             initial_load_completed.store(true, Ordering::Relaxed);
 
+                            // In Append mode, mark the dataset as ready immediately after the first message is received
+                            if is_append_mode_stream {
+                                self.update_component_status(status::ComponentStatus::Ready)
```

Comment:
I think we should probably do this for changes as well to mitigate https://github.com/spiceai/spiceai/issues/5309

Reply:
Updated 👍